lc56.py

```diff
@@ -1 +1,24 @@
+class Solution:
+    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
+        def isOverlap(a,b):
+            maxStart = max(a[0],b[0])
+            minFinish = min(a[1],b[1])
+            return (minFinish >=  maxStart) # overlap
+        
+        intervals.sort(key = lambda x:x[0])
```

Comment:
Technically a key is not needed here because list sorting will always compare the first index first. In case where you use have to use second element as a comparator I would recommend using things like itemgetter.